packages/app-bridge/src/react/useDocuments.ts

```diff
@@ -26,56 +21,22 @@ export const useDocuments = (appBridge: AppBridgeTheme) => {
 
     useEffect(() => {
         refetch();
-    }, [refetch]);
-
-    useEffect(() => {
-        const handleEventUpdates = (event: Event) => {
-            setDocuments((previousState) => {
-                const handler = actionHandlers[event.action] || actionHandlers.default;
-
-                return handler(previousState, event.document as Document);
-            });
-        };
-
-        const handleStandardDocumentEventUpdates = ({
-            action,
-            standardDocument,
-        }: {
-            action: EmitterAction;
-            standardDocument: Document | { id: number };
-        }) => handleEventUpdates({ action, document: standardDocument });
-
-        const handleLibraryEventUpdates = ({
-            action,
-            library,
-        }: {
-            action: EmitterAction;
-            library: Document | { id: number };
-        }) => handleEventUpdates({ action, document: library });
-
-        const handleLinkEventUpdates = ({ action, link }: { action: EmitterAction; link: Document | { id: number } }) =>
-            handleEventUpdates({ action, document: link });
 
-        const handleDocumentMoveEvent = (event: { action: 'update'; document: Document }) => handleEventUpdates(event);
-
-        window.emitter.on('AppBridge:GuidelineStandardDocumentAction', handleStandardDocumentEventUpdates);
-        window.emitter.on('AppBridge:GuidelineLibraryAction', handleLibraryEventUpdates);
-        window.emitter.on('AppBridge:GuidelineLinkAction', handleLinkEventUpdates);
-        window.emitter.on('AppBridge:GuidelineDocumentMoveAction', handleDocumentMoveEvent);
+        window.emitter.on('AppBridge:GuidelineDocumentAction', refetch);
 
         return () => {
-            window.emitter.off('AppBridge:GuidelineStandardDocumentAction', handleStandardDocumentEventUpdates);
-            window.emitter.off('AppBridge:GuidelineLibraryAction', handleLibraryEventUpdates);
-            window.emitter.off('AppBridge:GuidelineLinkAction', handleLinkEventUpdates);
-            window.emitter.off('AppBridge:GuidelineDocumentMoveAction', handleDocumentMoveEvent);
+            window.emitter.off('AppBridge:GuidelineDocumentAction', refetch);
```

Comment:
Why _again_ renaming the events back to just document?

Reply:
Yes, as a single event simplify things, and later when every entity is a truly separate entity we will create hooks per entity and events as well. Forgot about the terrific side, thanks for the reminder 👍 on it!